charts/jenkins/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ Use the following links to reference issues, PRs, and commits prior to v2.6.0.
 The change log until v1.5.7 was auto-generated based on git commits.
 Those entries include a reference to the git commit to be able to get more details.
 
+## 3.9.4
+
+Add _JAVA_OPTIONS to the README so proxy settings get picked by jenkins-plugin-cli
```

Comment:
please correct the variable name here and in PR title

Reply:
Done